src/main/java/core/web/filters/AuthenticationFilter.java

```diff
@@ -0,0 +1,50 @@
+package core.web.filters;
+
+import core.lib.Injector;
+import core.service.DriverService;
+import java.io.IOException;
+import javax.servlet.Filter;
+import javax.servlet.FilterChain;
+import javax.servlet.FilterConfig;
+import javax.servlet.ServletException;
+import javax.servlet.ServletRequest;
+import javax.servlet.ServletResponse;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+public class AuthenticationFilter implements Filter {
+    private static final Injector injector =
+            Injector.getInstance("core");
+    private static final String DRIVER_ID = "driver_id";
+    private final DriverService driverService =
+            (DriverService) injector.getInstance(DriverService.class);
+
+    @Override
+    public void init(FilterConfig filterConfig) throws ServletException {
+
+    }
+
+    @Override
+    public void doFilter(ServletRequest servletRequest,
+                         ServletResponse servletResponse, FilterChain chain)
+            throws IOException, ServletException {
+        HttpServletRequest req = (HttpServletRequest) servletRequest;
+        HttpServletResponse resp = (HttpServletResponse) servletResponse;
+        String url = req.getServletPath();
+        if (url.equals("/login") || url.equals("/drivers/add")) {
+            chain.doFilter(req, resp);
+            return;
```

Comment:
ideally, we can replace if-else construction with a Set, and  when you list or URLs grows larger it won't make this structure complicated